docs/using/tutorial/cluster.crt

```diff
@@ -0,0 +1 @@
+../../../vagrant/tutorial/credentials/cluster.crt
```

Comment:
Perhaps put a note in / near these files explaining how the files were generated.